client/src/components/App.jsx

```diff
@@ -3,6 +3,47 @@ import axios from 'axios';
 import styled from 'styled-components';
 import List from './List.jsx';
 
+const Wrapper = styled.div`
+    padding-top: 400px;
+    padding-right: 50px;
+    padding-bottom: 100px;
+    padding-left: 50px
+  `;
+
+const Top = styled.section`
+    display: flex;
+    justify-content: space-between;
+    padding-bottom: 24px; 
+`;
+const Right = styled.div`
+  align-items: center !important;
+  display: flex !important;
+  white-space: nowrap  
+`;
+const Title = styled.div`
+  text-align: left;
+  color: black;
+  padding-bottom: 0px;
+  font-weight: 600 !important;
+  font-size: 22px !important;
+  line-height: 26px
+  `;
+
+const Button = styled.span`
+  display: inline-flex;
+  appearance: none 
```

Comment:
I had a few issues where missing semocolons in css caused issues in my code. It may be worth adding them in.

Reply:
Thank you!